binder/src/main/java/io/grpc/binder/internal/BinderTransport.java

```diff
@@ -576,6 +578,32 @@ public BinderClientTransport(
         SecurityPolicy securityPolicy,
         InboundParcelablePolicy inboundParcelablePolicy,
         Attributes eagAttrs) {
+      this(
+          sourceContext,
+          targetAddress,
+          bindServiceFlags,
+          mainThreadExecutor,
+          executorServicePool,
+          offloadExecutorPool,
+          securityPolicy,
+          inboundParcelablePolicy,
+          eagAttrs,
+          BinderChannelCredentials.forDefault(sourceContext),
```

Comment:
Since this is an internal class, we shouldn't need to multiple constructors here. Just update the existing one?

Reply:
Unfortunately, the constructor of BinderClientTransport is public and Google3 has reference to it. So for now, we have to create a new constructor.  Since it is public, maybe 3rd party could use the constructor directly.  IMHO even we remove all the references to the old constructor in Google codebases, we still cannot remove the old constructor safely.

Reply:
There's no commitment to these internal APIs being stable, so breaking changes are explicitly fine here.

The google3 references are from before the open sourcing of this code, and we can update them with the import to g3.